modules/core/src/image/vpImageConvert.cpp

```diff
@@ -281,6 +281,86 @@ vpImageConvert::createDepthHistogram( const vpImage< uint16_t > &src_depth, vpIm
   }
 }
 
+/*!
+  Convert the input float depth image to a color depth image. The input
+  depth value is assigned a color value proportional to its frequency. The
+  alpha component of the resulting image is set to vpRGBa::alpha_default.
+  \param[in] src_depth : Input float depth image.
+  \param[out] dest_rgba : Output color depth image.
+*/
+void
+vpImageConvert::createDepthHistogram( const vpImage< float > &src_depth, vpImage< vpRGBa > &dest_rgba )
+{
+  dest_rgba.resize( src_depth.getHeight(), src_depth.getWidth() );
+  static uint32_t histogram[0x10000];
+  memset( histogram, 0, sizeof( histogram ) );
+
+  for ( unsigned int i = 0; i < src_depth.getSize(); ++i )
+    if(!std::isnan(src_depth.bitmap[i]))
```

Comment:
```suggestion
    if(!vpMath::isNaN(src_depth.bitmap[i]))
```

[std::isnan](https://en.cppreference.com/w/cpp/numeric/math/isnan) is C++11 feature.

Reply:
Done.